vars/dockerExecute.groovy

```diff
@@ -47,15 +47,15 @@ import groovy.transform.Field
     /**
       * The registry used for pulling the docker image, if left empty the default registry as defined by the `docker-commons-plugin` will be used.
       */
-    'dockerRegistry',
+    'dockerRegistryUrl',
     /**
       * The credentials for the docker registry. If left empty, images are pulled anonymously.
       */
     'dockerRegistryCredentials',
     /**
-      * Same as `dockerRegistry`, but for the sidecar. If left empty, `dockerRegistry` is used instead.
+      * Same as `dockerRegistryUrl`, but for the sidecar. If left empty, `dockerRegistryUrl` is used instead.
       */
-    'dockerSidecarRegistry',
+    'dockerSidecarRegistryUrl',
```

Comment:
`dockerSidecarRegistryUrl` and `dockerSidecarRegistryCredentials` should be `sidecarRegistry..` as the image is also just names `sidecarImage`.

Reply:
Ok.